variables.tf

```
@@ -0,0 +1,45 @@
+### Common
+variable "aws_region" {
+  description = "AWS Region"
+  type        = string
+  default     = "us-east-1"

```

Se usarmos como default uma string vazia, o provider da AWS usará a variável `AWS_REGION` ou o que encontrar na configuração do AWS CLI. Acredito ser mais interessante do que usar um valor padrão aqui, até porque a região pode estar inerentemente atrelada a um deployment / environment.

```suggestion
  default     = ""
```